app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -147,6 +149,11 @@ class EventsFragment : Fragment() {
         return rootView
     }
 
+    private fun showNoEventText(show: Boolean) {
```

Comment:
`showNoEventText` you are also showing an image. This is a bit confusing

Reply:
should I name it to showNoEventPannel or showNoEventLayout